AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/config/CustomMessages.java

```diff
@@ -1,246 +1,312 @@
+        addDefault("Info.tpallRequestSent", "<prefix> <gray>Successfully sent a teleport request to <aqua><amount></aqua>player(s)!");
+        addDefault("Info.teleportedToLoc", "<prefix> <gray>Successfully teleported you to <aqua>x</aqua>, <aqua>y</aqua>, <aqua>z</aqua>! (Yaw: <aqua>yaw</aqua>, Pitch: <aqua>pitch</aqua>, World: <aqua>world</aqua>)");
+        addDefault("Info.teleportedToLocOther", "<prefix> <gray>Successfully teleported <aqua><player></aqua>to <aqua>x</aqua>, <aqua>y</aqua>, <aqua>z</aqua>! (Yaw: <aqua>yaw</aqua>, Pitch: <aqua>pitch</aqua>, World: <aqua>world</aqua>)");
+        addDefault("Info.movedWarp", "<prefix> <gray>Moved <aqua><warp></aqua>to your current location!");
+        addDefault("Info.movedHome", "<prefix> <gray>Moved home <aqua><home></aqua>to your current location!");
+        addDefault("Info.movedHomeOther", "<prefix> <gray>Moved <aqua><player>'s </aqua> home <aqua><home></aqua>to your location!");
+        addDefault("Info.setMainHome", "<prefix> <gray>Made <aqua><home></aqua>your main home!");
+        addDefault("Info.setAndMadeMainHome", "<prefix> <gray>Set <aqua><home></aqua>at your current location and made it your main home!");
+        addDefault("Info.setMainHomeOther", "<prefix> <gray>Made <aqua><home> <player></aqua>'s main home!");
+        addDefault("Info.setAndMadeMainHomeOther", "<prefix> <gray>Set <aqua><home></aqua>for <aqua><player></aqua>at your current location and made it their main home!");
+        addDefault("Info.mirroredSpawn", "<prefix> <gray>Mirrored <aqua>from</aqua>'s spawnpoint to <aqua>spawn</aqua>!");
+        addDefault("Info.setMainSpawn", "<prefix> <gray>Set the main spawnpoint to <aqua>spawn</aqua>! All players will teleport there if there are no overriding spawns/permissions.");
+        addDefault("Info.removedSpawn", "<prefix> <gray>Removed the spawnpoint <aqua>spawn</aqua>!");
+        addDefault("Info.setSpawnSpecial", "<prefix> <gray>Set spawnpoint <aqua>spawn</aqua>!");
+        addDefault("Info.importStarted", "<prefix> <gray>Starting import from <aqua>plugin</aqua>...");
+        addDefault("Info.importFinished", "<prefix> <gray>Finished import from <aqua>plugin</aqua>!");
+        addDefault("Info.exportStarted", "<prefix> <gray>Starting export to <aqua>plugin</aqua>...");
+        addDefault("Info.exportFinished", "<prefix> <gray>Finished export to <aqua>plugin</aqua>!");
+        addDefault("Info.paymentItems", "<prefix> <gray>You have paid <aqua><amount> <type>(s)</aqua> for that teleport!");
+        addDefault("Info.updateInfo", """
+            <prefix> [&7AdvancedTeleport has an update available!
+            Click/hover over this text for more information.]
+            (&bCurrent Version » &7<version>|&bNew Version <dark_gray> &7{new-version}|&bTitle <dark_gray> &7<title>
+            |https://www.spigotmc.org/resources/advancedteleport.64139/)
+        """.trim());
```

Comment:
Same for this but if it's not clear what the format intended is I can change that myself.